src/QsCompiler/CompilationManager/Utils.cs

```diff
@@ -183,7 +183,7 @@ public static (List<T>, List<T>) Partition<T>(this IEnumerable<T> collection, Fu
         /// Projects each element of a sequence into a new form and discards null values.
         /// </summary>
         /// <remarks>Overload for reference types.</remarks>
-        internal static IEnumerable<TResult> SelectNotNull<TSource, TResult>(
+        public static IEnumerable<TResult> SelectNotNull<TSource, TResult>(
```

Comment:
Why were the `SelectNotNull` methods changed from `internal` to `public`?

Reply:
They're needed by the language server, although ultimately I think they need to live in Transformations since that is the one C# project that all other C# projects have a reference to (see https://github.com/microsoft/qsharp-compiler/pull/644#discussion_r494515713)